roles/edpm_nova/templates/nova_compute.json.j2

```diff
@@ -29,5 +29,10 @@
         "/etc/nvme:/etc/nvme",
         "/var/lib/openstack/config/ceph:/var/lib/kolla/config_files/ceph:ro",
         "/etc/ssh/ssh_known_hosts:/etc/ssh/ssh_known_hosts:ro"
+{% if edpm_nova_extra_bind_mounts | length > 0 %}
+    {% for bind_mount in edpm_nova_extra_bind_mounts %}
```

Comment:
Another nit: I'm a similar situation, I wanted to suppress the j2 statements from inserting blank lines and found I could append/prepend '-' dashes to yield a clean result. Not sure if that's a factor here, but offer it for your consideration.

Reply:
yep i can add the - to make it trim space

Reply:
im going to finish shortly today so ill likely respin this tomororow

Reply:
im also testing this
when i have used that in go tempelates it was tricky to get right with lot of nesting but with just this simple templating i think its probably doable
ill include this when updating for the if